x/io/int.go

```diff
@@ -4,15 +4,26 @@ import (
 	"encoding/binary"
 	"errors"
 	"io"
-	"math"
-	"math/bits"
 )
 
 var (
 	errBufferTooSmall = errors.New("buffer too small")
 	errValueTooLarge  = errors.New("value too large")
 )
 
+// Used for calculating the number of bytes required to encode a varint.
+const (
+	one   = (1 << 7) / 2
```

Comment:
Use `>>1` instead of `/ 2`

Reply:
Ah, I figured the compiler would optimize this away anyways and it wouldn't be calculated at runtime. So I went w/ what is easier to read.

Can change though.